tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_hashes/fresh_system.pass.sh

```diff
@@ -4,6 +4,6 @@
 
 # verify (-V) all installed (-a) packages digests,
 # if digest differs from package metadata, then attribute '5' is printed
-result=$(rpm -Va | grep '^..5')
+result=$(rpm -Va | grep '^..5.* /(bin|sbin|lib|lib64|usr)/')
```

Comment:
@mildass For me, this needs `-E` for grep to list the filenames.

Reply:
@yuumasato Yes, sorry. I forgot it and I didn't think that's wrong, because test passed (but it passed because of empty grep output).